Helpers.gs

```diff
@@ -701,7 +705,7 @@ function parseAttendeeResp(veventString){
  * Will return 0 by default.
  *
  * @param {string} notificationString - The string to parse
- * @return {number} The notification time in seconds
+ * @return {number} The notification time in minutes
  */
 function parseNotificationTime(notificationString){
```

Comment:
What was the rationale for changing this to minutes? Is it not possible for a notification trigger to contain `__S` or did we just decide to not support it since Google Calendar doesn't support a number of seconds for a notification?

Reply:
I'm sure a trigger could include seconds (even though i have not found a definition of all valid options) but google will not add these event.

Triggers like **-PT30M30S** currently throw an error at _Calendars.insert_, with the change it would just omit the seconds and add a reminder 30min prior to the event, that's as close as we can get..

Reply:
Sounds like a good assessment.